### ISSUE TITLE
Jump between civic centers with a hotkey

### ISSUE THREAD

**Issue by LangLangBart** [LangLangBart/boonGUI#28]
A nice quality of life feature from AOE2 is the possibility to go through all civic centers with a hotkey. As an active player you would only go through your own (no allies) and as an observer through all civic centers.

**Comment by sl5net** [comment]
yes as active player i use jump to a building by 1. selecting it and then 2. jump to railway route. both with hotkeys. 

in replay i not missed a toggle hotkey for jump between all CC's. 

But, yes, my idea is that if if CC is already selected and i press select CC hotkey again then it should jump to next CC or select next CC (i could then jump to railway point again with same hotkey that i use for select railway points).

so we dont need to wast a new hotkey (prevent hotkey war).

**Comment by LangLangBart** [comment]
AOE4 has it set to `Ctrl+H`
<img src="https://ttm.sh/iM9.30.jpg" width="600">

Setting up a hotkey is doable, cycling through all cc's for active players and observers is currently the roadblock.

**Comment by StanleySweet** [comment]
Might have something of interest to you here https://trac.wildfiregames.com/ticket/1899
